### DISCUSSION TITLE
Seeing status of referenced tasks in requests

### DISCUSSION THREAD

**Discussion by BioMedGPH** [Samedis-care/samedis-care-community#88] [Shipped - Live on production]
I noticed that when tasks are created from a user request they are listd as referenced tasks. It would be great to see here also the status of the tasks (open pending done) so in a quick glance we can see if all referenced tasks are done and if the request can be finished.

**Comment by YvesR** [comment]
You do mean other created tasks for the same inventory that were created out of the scope of the incident request?

**Reply by BioMedGPH** [reply]
no I mean in scope of the incident request.
For instance, a user reports some issue. On inspection it appears there are actually 2 malfunctions causing the issue that both need repair. Then these 2 repair tasks will show as referenced task in the user request. For us it would be great to see if those 2 tasks then would also show their status inside the user request, so at a glance we can see that all malfunctions leading to this user requests are addressed. This is relevant as sometimes different repairs have a very different timeline.

**Reply by YvesR** [reply]
Hello, do you mean something like this?

<img width="464" alt="example" src="https://github.com/Samedis-care/samedis-care-community/assets/4928882/e340898c-42e3-4fd6-b04d-a6db18a1d049">

**Comment by BioMedGPH** [comment]
Hi Yves,

Yes something like that looks good. 

regards,
Arjan

Op 19 jun. 2024, om 13:05 heeft Yves Rausch ***@***.***> het volgende geschreven:


Hello, do you mean something like this?

example.png (view on web) <https://github.com/Samedis-care/samedis-care-community/assets/4928882/e340898c-42e3-4fd6-b04d-a6db18a1d049>

**Reply by YvesR** [reply]
we will implement this.
